rltk/evaluation/evaluation.py

```diff
@@ -140,3 +140,40 @@ def plot_roc(self):
             'x': 'false_positives',
             'y': 'true_positives'
         }], auc_params=[0.05, 0.95, True])
+
+    def plot_features(self, df, f1, f2):
```

Comment:
- please add docstring.
- can you make them to show as subgraphs in one big graph? ideally, this function should return `plt` and let user do `show()`, I don't think it's elegant to show one by one (maybe we can just return a list of plt objects).